docs/source/07-02-troubleshooting-tips.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Troubleshooting Tips
+permalink: /troubleshooting-tips/
+---
+
+## I've installed Talk but I can't see the comment stream appear on my articles
+
+* Make sure you've adding the correct domains to your Permitted Domains in Configure > Tech Settings
+* Make sure you've correctly added the embed via your CMS to your article pages
```

Comment:
Link off to `/talk/integrating/asset-management/`

Reply:
Fixed!